SourceCode/Builds/vs22_compile.md

```diff
@@ -2,14 +2,31 @@
 
 ## Prerequisites
 
-1. **Install Visual Studio 2022**
-   - Ensure that the necessary C++ development components, including MFC, are installed.
-
-2. **Obtain the C&C Generals Source Code**
-   - Clone or download the source code repository: [Super Hackers Generals Game Code](https://github.com/TheSuperHackers/GeneralsGameCode.git).
-
-3. **Install C&C Generals (Steam Version)**
-   - The game installation is required to access the necessary asset files.
+- **Install Visual Studio 2022**  
+  Ensure that the necessary C++ development components, including MFC, are
+  installed.
+
+- **Obtain the C&C Generals Source Code**  
+  Clone or download the source code repository:  
+  [jmarshall2323 VS2022 Fork](https://github.com/jmarshall2323/CnC_Generals_Zero_Hour.git).
+
+- **Install C&C Generals**  
+  The game installation is required to access the necessary asset files.
+
+- **Download the necessary SDKs as needed**  
+  3ds max sdk: [3ds-max sdk](https://archive.org/details/maxsdk-4.2.0.85).  
+  Download and extract the contents of the zip file to the folder  
+  `/Code/Libraries/max4sdk`
+  Don't forget to add into the build projects
+  ( so far `\toolchain\max2w3d`  and `\toolchain\pluglib`)
+  the paths to include and lib folders.
```

Comment:
I do not understand this.

Reply:
ok, those toolchain projects need the sdk mentioned above. and since it is on a different folder, you need to setup the include and lib paths correctly so the compiler find those files